ocs_ci/ocs/platform_nodes.py

```diff
@@ -1605,6 +1606,16 @@ def restart_nodes_teardown(self):
         for bm in bms:
             bm.session.close()
 
+    def restart_nodes_by_stop_and_start(self, nodes, force=True):
+        """
+        Restart Bare Metal Nodes
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            force (bool): True for force BM stop, False otherwise
+        """
+        self.baremetal.restart_baremetal_machines(nodes, force=force)
```

Comment:
why is this method same for both restart_nodes and restart_nodes_by_stop_and_start?

Reply:
in which case the function should be just an alias, one of the def is not needed.

Reply:
Yeah we could simply this by having an argument for `stop_and_start` and have one method handle both cases.